csvcubed/csvcubed/readers/V1_0/preconfiguredtemplates.py

```diff
@@ -0,0 +1,46 @@
+"""
+Pre-configured JSON Templates
+-----------------------------
+
+Functionality to help augment JSON files with configuration from some pre-configured templates.
+"""
+import json
+import logging
+import requests
+from typing import Dict, Any
+
+def apply_preconfigured_values_from_template(column_config: Dict[str, Any], version_module_path: str) -> None:
+    # if column_config doesn't have the `from_template` property, just terminate the function now
+    if "from_template" not in column_config:
+        return
+        
+    # if column_config has `from_template` property, extract that value
+    # remove the `from_template` property
+    else:
```

Comment:
You don't need an `else` statement here since the other condition exits from the function.